cocotb/share/makefiles/simulators/Makefile.icarus

```diff
@@ -62,21 +60,10 @@ $(SIM_BUILD)/sim.vvp: $(SIM_BUILD) $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS)
 
 # Execution phase
 
-ifeq ($(OS),Msys)
-
-OLD_PATH := $(shell echo "$(PATH)" | sed 's/(/\\(/g' | sed 's/)/\\)/g' | sed 's/ /\\ /g')
-LIB_LOAD = PATH=$(OLD_PATH):$(LIB_DIR)
-NEW_PYTHONPATH:=$(shell python -c "import sys, os; print(':'.join(['/'+dir.replace(os.sep,'/').replace(':','') for dir in sys.path]))")
-
-else
-LIB_LOAD := LD_LIBRARY_PATH=$(LIB_DIR):$(LD_LIBRARY_PATH)
-NEW_PYTHONPATH := $(PYTHONPATH)
-endif
-
 $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/sim.vvp $(CUSTOM_SIM_DEPS) $(COCOTB_LIBS) $(COCOTB_VPI_LIB)
 	-@rm -f $(COCOTB_RESULTS_FILE)
 
-	PYTHONPATH=$(LIB_DIR):$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
+	PYTHONPATH=$(PYTHONPATH) MODULE=$(MODULE) \
```

Comment:
I'm not sure why, but with Windows 10 WSL (Ubuntu 18.04 LTS) removing `LD_LIBRARY_PATH` causes `iverilog` to be unable to find some of the libraries, causing a `dlopen()` error.

Reply:
This is very strange configuration. It looks like Windows WSL does not respect `Library runpath` 
One can go around this bay setting:  `LD_LIBRARY_PATH=$(cocotb-config --prefix)/cocotb/libs`

Reply:
Once this is in, it'd be nice to document that workaround somewhere (on a wiki page?)

Reply:
did you tried WSL 2?

Reply:
WSL1 is dead, let's add a doc/wiki entry somewhere (probably sufficient to say "it's not supported").

Reply:
@garmin-mjames I know what is wrong will try to make PR tomorrow. It is a more general issue.

Reply:
This should fix this https://github.com/cocotb/cocotb/pull/1606